po/Makevars

```diff
@@ -8,7 +8,7 @@ subdir = po
 top_builddir = ..
 
 # These options get passed to xgettext.
-XGETTEXT_OPTIONS = --keyword=_ --keyword=N_
+XGETTEXT_OPTIONS = --from-code=UTF-8 --keyword=_ --keyword=N_ --keyword=C_:1c,2 --keyword=NC_:1c,2 --keyword=g_dngettext:2,3 --add-comments
```

Comment:
I'd append '--keyword=Name --keyword=Comment --keyword' to XGETTEXT_OPTIONS so that xgettext won't extract 'Icon' line in *.desktop files and also add the two lines:
```
# The last argument --keyword without keyword prevents xgettext from
# extracting translatable strings from "Icon" in *.desktop files.
```

Reply:
Humm, I don't see it extracting the `Icon` names from desktop files at the moment. Does it happened on your machine?

Reply:
I use Fedora 29 gettext 0.19.8.1 and the result is:

```
#: setup/python2/ibus-setup-anthy.desktop.in.in:6
#: setup/python3/ibus-setup-anthy.desktop.in.in:6
msgid "ibus-anthy"
msgstr ""
```

If my suggestion is not applied, the icon name "ibus-anhty" is extracted in my box.

Reply:
I run `make ibus-anthy.pot` in ibus-anthy/po.

Reply:
Oh, I didn't notices that `ibus-anthy.pot` was actually stored in the repo. What's why I couldn't reproduce on my machine. Changed, thanks!